getting-started/orientation/index.md

```diff
@@ -485,7 +590,6 @@ The string is the branch and commit you are running, for those who speak [Git](h
 
 ## Quitting OpenSpace
 
-To quit OpenSpace you have three options:
+To quit OpenSpace you have two options:
 1. Use your Operating System's quit button in the top of the window.
-2. Press {kbd}`Esc` and you will see a three-second timer. At the end of that three seconds, OpenSpace will close and quit. However, if you errantly quit, you can hit {kbd}`Esc` again to cancel the quit.
```

Comment:
We still have the `CTRL+Q` that will close the application, so there are still three options left